File: frontend/universidad/src/app/models/facultad.ts
export interface Facultad {
  facultadId: number;
  nombre: string;
  descripcion?: string;
  ubicacion?: string;
  decano?: string;
  fechaRegistro: Date;
  activo: boolean;
}

export interface CreateFacultad {
  nombre: string;
  descripcion?: string;
  ubicacion?: string;
  decano?: string;
}

export interface UpdateFacultad {
  nombre?: string;
  descripcion?: string;
  ubicacion?: string;
  decano?: string;
  activo?: boolean;
}

File: backend/UniversidadAPI/Models/Facultad.cs
namespace UniversidadAPI.Models
{
    public class Facultad
    {
        public int FacultadId { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string? Descripcion { get; set; }
        public string? Ubicacion { get; set; }
        public string? Decano { get; set; }
        public DateTime FechaRegistro { get; set; }
        public bool Activo { get; set; }
    }
}